op-batcher/batcher/driver.go

```diff
@@ -344,7 +343,43 @@ func (l *BatchSubmitter) publishStateToL1(queue *txmgr.Queue[txData], receiptsCh
 	}
 }
 
-// publishTxToL1 queues a single tx to be published to the L1
+// clearState clears the state of the channel manager
+func (l *BatchSubmitter) clearState(ctx context.Context) {
+	l.Log.Info("Clearing state")
+	defer l.Log.Info("State cleared")
+
+	// Attempt to set the L1 safe origin and clear the state, if fetching fails -- fall through to an infinite retry
+	l1SafeOrigin, err := l.safeL1Origin(ctx)
+	if err != nil {
+		l.state.Clear(l1SafeOrigin)
+		return
+	}
+
+	l.Log.Info("failed to query L1 safe origin, will retry", "err", err)
+
+	tick := time.NewTicker(5 * time.Second)
+	defer tick.Stop()
+
+	for {
+		select {
+		case <-tick.C:
+			l1SafeOrigin, err := l.safeL1Origin(ctx)
+			if err != nil {
+				l.Log.Warn("Failed to query L1 safe origin, will retry", "err", err)
+			} else {
+				l.Log.Info("Clearing state with safe L1 origin", "origin", l1SafeOrigin)
+				l.state.Clear(l1SafeOrigin)
+				return
+			}
```

Comment:
You might want to move this into its own anonymous function (maybe returning a bool for success) above and call it before the loop, so you don't repeat the log lines.

Right now, they are inconsistent. First success doesn't log, but in the loop info-logs. First fail info-logs, retry fail warn-logs.